puppet/services/neutron-honeycomb-agent.yaml

```diff
@@ -0,0 +1,84 @@
+heat_template_version: newton
+
+description: >
+  OpenStack Neutron Honeycomb agent configured with Puppet
+
+parameters:
+  ServiceNetMap:
+    default: {}
+    description: >
+      Mapping of service_name -> network name. Typically set via
+      parameter_defaults in the resource registry. This mapping overrides those
+      in ServiceNetMapDefaults.
+    type: json
+  DefaultPasswords:
+    default: {}
+    type: json
+  EndpointMap:
+    default: {}
+    description: Mapping of service endpoint -> protocol. Typically set
+                 via parameter_defaults in the resource registry.
+    type: json
+  HoneycombUser:
+    description: Username for Honeycomb agent
+    type: string
+    default: 'admin'
+  HoneycombPassword:
+    description: Password for Honeycomb agent
+    type: string
+    hidden: true
+    default: 'admin'
+  HoneycombRestPort:
+    description: Port for Honeycomb REST interface to listen on.
+    type: number
+    default: 8183
+  HoneycombNetconfSshPort:
+    description: Honeycomb Netconf SSH binding port.
+    type: number
+    default: 2831
+  OpenDaylightUsername:
+    default: 'admin'
+    description: The username for the opendaylight server.
+    type: string
+  OpenDaylightPassword:
+    default: 'admin'
+    type: string
+    description: The password for the opendaylight server.
+    hidden: true
+  OpenDaylightPort:
+    default: 8081
+    description: Opendaylight service port
+    type: number
+resources:
+
+  NeutronBase:
+    type: ./neutron-base.yaml
+    properties:
+      ServiceNetMap: {get_param: ServiceNetMap}
+      DefaultPasswords: {get_param: DefaultPasswords}
+      EndpointMap: {get_param: EndpointMap}
+
+outputs:
+  role_data:
+    description: Role data for Honeycomb agent service.
+    value:
+      service_name: neutron_honeycomb_agent
```

Comment:
I'm kind of at odds over the name.  It isn't really a neutron agent, it is an ODL agent, but it kind of lives under the Neutron umbrella because ODL does.. what about neutron_odl_honeycomb_agent?  I'm worried about the name getting too long as well by spelling out opendaylight.